src/Helpers/photoDome.ts

```diff
@@ -55,7 +55,7 @@ export class PhotoDome extends TextureDome<Texture> {
     }
 
     protected _initTexture(urlsOrElement: string, scene: Scene, options: any): Texture {
-        return new Texture(urlsOrElement, scene, true, !this._useDirectMapping, undefined, undefined, (message, exception) => {
+        return new Texture(urlsOrElement, scene, options.noMipMap, !this._useDirectMapping, undefined, undefined, (message, exception) => {

```

Comment:
if it was true before we should set it to true by default right?

Reply:
yes! will do.

Reply:
Converted to generateMipMaps, and with that kept the default behavior for both video and photo dome